modules/account-lib/src/coin/eth/transactionBuilder.ts

```diff
@@ -400,8 +400,8 @@ export class TransactionBuilder extends BaseTransactionBuilder {
       this.validateValue(new BigNumber(fee.eip1559.maxFeePerGas));
       this.validateValue(new BigNumber(fee.eip1559.maxPriorityFeePerGas));
     }
-    if (fee.fee) {
-      this.validateValue(new BigNumber(fee.fee));
+    if (fee.gasPrice) {
```

Comment:
shouldn't we still validate `fee.fee` ?

Reply:
The first line of this function validates fee.fee so I modified this one to gasPrice but maybe the other one should also be in an if.